docs/product-manuals/modeler/cloud-modeler/launch-cloud-modeler.md

```diff
@@ -3,6 +3,6 @@ id: launch-cloud-modeler
 title: Launch Cloud Modeler
 ---
 
-Since the Cloud Modeler is embedded in the Camunda Cloud Console, it does not need to be started separately. Go to the tab **BPMN Diagrams** and click the button **Create New Diagram** or click on an **existing diagram** to jump to the Modeler.
+Since the Cloud Modeler is embedded in the Camunda Cloud Console, it does not need to be started separately. Go to the tab **Diagrams** and click the button **Create New Diagram** or click on an **existing diagram** to jump to the Modeler.
```

Comment:
Maybe just a minor thing: but there are no **tabs** anymore on the overview page. There is just the navbar with the navigation items.